doc/org.eclipse.cdt.doc.user/.asciidoctorconfig.adoc

```diff
@@ -0,0 +1,13 @@
+// +++++++++++++++++++++++++++++++++++++++++++++++++++++++
+// +  Initial AsciiDoc editor configuration file - V1.0  +
+// ++++++++++++++++++++++++++++++++++++++++++++++++++++++
+// 
+// Did not find any configuration files, so creating this at project root level.
+// If you do not like those files to be generated - you can turn it off inside Asciidoctor Editor preferences.
+// 
+// You can define editor specific parts here.
+// For example: with next line you could set imagesdir attribute to subfolder "images" relative to the folder where this config file is located.
+// :imagesdir: {asciidoctorconfigdir}/images
+// 
+// For more information please take a look at https://github.com/de-jcup/eclipse-asciidoctor-editor/wiki/Asciidoctor-configfiles
+:imagesdir: {asciidoctorconfigdir}/images
```

Comment:
We instruct the AsciiDoc editor to look for images in the `images` folder of the project. Example:
![image](https://github.com/eclipse-cdt/cdt/assets/45816183/08ee11f6-7560-4a0f-9908-081181de2057)